prombench/manifests/prombench/benchmark/3b_prometheus-test_deployment.yaml

```diff
@@ -157,12 +164,19 @@ spec:
         args: [
           "--web.external-url=http://{{ .DOMAIN_NAME }}/{{ .PR_NUMBER }}/prometheus-release",
           "--storage.tsdb.path=/prometheus",
+          "--storage.tsdb.no-lockfile",
+          "--storage.tsdb.retention.time=30m",
+          "--storage.tsdb.wal-compression",
+          "--storage.tsdb.min-block-duration=10m",
+          "--storage.tsdb.max-block-duration=10m",
           "--web.console.libraries=/etc/prometheus/console_libraries",
           "--web.console.templates=/etc/prometheus/consoles",
+          "--web.enable-lifecycle",
           "--config.file=/etc/prometheus/prometheus.yml",
         # JSON log format is needed for GKE to display log levels correctly.
           "--log.format=json",
-          "--log.level=debug"
+          "--log.level=debug",
+          "--enable-feature=exemplar-storage",
```

Comment:
why did you add `--export.user-agent-mode=kubectl` 

https://github.com/GoogleCloudPlatform/prometheus-engine/blob/722ee81b87348557d7274be0dccfa14c3987b9af/cmd/operator/deploy/operator/10-collector.yaml#L109C11-L109C43